.github/actions/orca/orca-python-horovod-tf2-py37-spark3-action/pr-test/action.yml

```diff
@@ -16,6 +16,8 @@ runs:
         # install orca
         pip install -i https://pypi.org/simple python/orca/src/dist/bigdl_orca*-py3-none-manylinux1_x86_64.whl
```

Comment:
No need to `pip uninstall bigdl-orca bigdl-orca-spark3` here. All the packages are removed in setting up env. 
e.g. https://github.com/intel-analytics/BigDL/blob/main/.github/actions/orca/setup-env/setup-horovod-tf2-py37/action.yml#L28